ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -202,3 +202,8 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-e2e
+- as: e2e-metal-ipi-nightly
```

Comment:
Are there other jobs like this for other platforms? I don't see them (but this repo is YAML soup so I might be missing it)

Release jobs would be more valuable in the periodics under `./ci-operator/jobs/openshift/release`